src/sec_extreme_values.mbx

```diff
@@ -25,13 +25,13 @@
     </aside>
 
     <p>Consider <xref ref="fig_extreme">Figure</xref>.
-      The function displayed in (a) has a maximum, but no minimum, as the interval over which the function is defined is open. In (b), the function has a minimum, but no maximum; there is a discontinuity in the <q>natural</q> place for the maximum to occur. Finally, the function shown in (c) has both a maximum and a minimum; note that the function is continuous and the interval on which it is defined is closed.</p>
+      The function displayed in <xref ref="fig_extreme1" /> has a maximum, but no minimum, as the interval over which the function is defined is open. In <xref ref="fig_extreme2" />, the function has a minimum, but no maximum; there is a discontinuity in the <q>natural</q> place for the maximum to occur. Finally, the function shown in <xref ref="fig_extreme3" /> has both a maximum and a minimum; note that the function is continuous and the interval on which it is defined is closed.</p>
 
-    <figure xml:id="fig_extreme" >
```

Comment:
It seems to me that this figure tag was extra? It is wrapped around a side by side. You can see what it looks like in the current version, Figure 3.1.2. [http://spot.pcc.edu/math/APEXCalculus/sec_extreme_values.html](url)

@davidfarmer